sources/models/src/aws-k8s-1.22-nvidia/defaults.d/95-oci-defaults.toml

```diff
@@ -0,0 +1 @@
+../../../shared-defaults/oci-defaults.toml
```

Comment:
It's a minor thing, but I'd prefer to have these in the 75-79 or 85-89 range, together with `??-oci-hooks.toml`, rather than at the end of the 90 range. That way all the OCI related defaults share the same range.

Reply:
Symlinks renamed in latest force push